src/resources/BreadCrumbResource.ts

```diff
@@ -1,11 +1,11 @@
 import { BreadcrumbIF } from '@/interfaces'
 
-export const HomeBreadCrumb: BreadcrumbIF = {
-  text: 'Business Registry Home',
-  href: `${sessionStorage.getItem('BUSINESSES_URL')}`
+export const DashboardHomeBreadcrumb: BreadcrumbIF = {
+  text: 'BC Registries Dashboard',
+  href: `${sessionStorage.getItem('BCROS_HOME_URL')}dashboard`
 }
 
-export const DashboardBreadcrumb: BreadcrumbIF = {
+export const RegistryTableBreadcrumb: BreadcrumbIF = {
```

Comment:
Would you mind renaming these to, say, "BcRegistriesDashboadBreadcrumb" and "MyBusinessRegistryDashboard"?

This may seem like a fickle request but even today there is much confusion on what different pages are called, so I think if the names are the same as their title then it's quite a bit clearer :)

Reply:
Design has pushed that there is only one 'dashboard' per se, so i was trying to avoid calling this a dashboard. But we can update this. Will take a bit 😆  
This effects all the PR's of course for consistency.

Reply:
All good though, after talking to Patrick we are going to update the name of the new ENV variable :)

Reply:
So i'm landing on `RegistryHomeDashboardBreadcrumb` and `MyBusinessRegistryTBreadcrumb`
Are we ok with that?

Reply:
This has been decided in RC discussion. Closing.